docker-compose.yml

```diff
@@ -1,49 +1,102 @@
-      - IPFS_API_URL=ipfs:5001
+
+      - ANCHOR_CONTROLLER_ENABLED=false
+      - ANCHOR_EXPIRATION_PERIOD=0
+      - ANCHOR_SCHEDULE_EXPRESSION=0/1 * * * ? *
+
       - BLOCKCHAIN_CONNECTOR=ethereum
-      - ETH_NETWORK=ganache
+      - CERAMIC_API_URL=http://host.docker.internal:7007
       - ETH_GAS_LIMIT=4712388
       - ETH_GAS_PRICE=100000000000
-      - ETH_RPC_HOST=http://ganache
-      - ETH_RPC_PORT=8545
-      - ETH_WALLET_PK=0x06dd0990d19001c57eeea6d32e8fdeee40d3945962caf18c18c3930baa5a6ec9
+      - ETH_NETWORK=ropsten
+      - ETH_OVERRIDE_GAS_CONFIG=false
+      - ETH_WALLET_PK=0x16dd0990d19001c50eeea6d32e8fdeef40d3945962caf18c18c3930baa5a6ec9
+
+      - IPFS_API_URL=ipfs:5001
+      - IPFS_API_TIMEOUT=10000
+
       - LOG_LEVEL=debug
       - LOG_TO_FILES=true
-      - TYPEORM_CONNECTION=sqlite
-      - TYPEORM_HOST=localhost
+      - LOG_PATH=/usr/local/var/log
+      - MERKLE_DEPTH_LIMIT=0
+
+      - TYPEORM_CONNECTION=postgres
+      - TYPEORM_DATABASE=anchor_db
+      - TYPEORM_HOST=database
+      - TYPEORM_PORT=5432
       - TYPEORM_USERNAME=root
       - TYPEORM_PASSWORD=root
-      - TYPEORM_DATABASE=./anchor.db.sq3
-      - TYPEORM_PORT=5432
+
+      - TYPEORM_ENTITIES=build/models/**/*.js
+      - TYPEORM_SUBSCRIBERS=build/subscribers/**/*.js
+      - TYPEORM_LOGGING=false
       - TYPEORM_SYNCHRONIZE=true
-      - TYPEORM_LOGGING=true
+
+      - VALIDATE_RECORDS=false
+    volumes:
+      - "/usr/local/var/log:/usr/local/var/log"
+    depends_on:
+      - ipfs
+    ports:
+      - "8081:8081"
+    networks:
+      - internal
+
+  cas_anchor:
+    build: .
+    restart: unless-stopped
+    environment:
+      - NODE_ENV=dev
+      - APP_MODE=anchor
+      - APP_PORT=8081
+      - ANCHOR_CONTROLLER_ENABLED=false
+      - ANCHOR_EXPIRATION_PERIOD=0
+      - ANCHOR_SCHEDULE_EXPRESSION=0/1 * * * ? *
+      - BLOCKCHAIN_CONNECTOR=ethereum
+      - CERAMIC_API_URL=http://host.docker.internal:7007
+      - ETH_GAS_LIMIT=4712388
+      - ETH_GAS_PRICE=100000000000
+      - ETH_NETWORK=ropsten
+      - ETH_OVERRIDE_GAS_CONFIG=false
+      - ETH_WALLET_PK=0x16dd0990d19001c50eeea6d32e8fdeef40d3945962caf18c18c3930baa5a6ec9
+      - IPFS_API_URL=ipfs:5001
+      - IPFS_API_TIMEOUT=10000
+      - LOG_LEVEL=debug
+      - LOG_TO_FILES=true
+      - LOG_PATH=/usr/local/var/log
+      - MERKLE_DEPTH_LIMIT=0
+
+      - TYPEORM_CONNECTION=postgres
+      - TYPEORM_DATABASE=anchor_db
+      - TYPEORM_HOST=database
+      - TYPEORM_PORT=5432
+      - TYPEORM_USERNAME=root
+      - TYPEORM_PASSWORD=root
+
       - TYPEORM_ENTITIES=build/models/**/*.js
+      - TYPEORM_SUBSCRIBERS=build/subscribers/**/*.js
+      - TYPEORM_LOGGING=false
+      - TYPEORM_SYNCHRONIZE=true
+
+      - VALIDATE_RECORDS=false
     volumes:
       - "/usr/local/var/log:/usr/local/var/log"
     depends_on:
       - ipfs
-      - ganache
     networks:
       - internal
+
   ipfs:
-    image: ipfs/go-ipfs:latest
+    image: ceramicnetwork/js-ipfs-ceramic
```

Comment:
I guess you're holding off to make the switch to ipfs-daemon for a later patch?

Reply:
Yup